script.js

```diff
@@ -2,57 +2,49 @@ let startBtn = document.querySelector("#startButton");
 startBtn.addEventListener("click", playGame);
 //ash says we can use this as a global object
 let gameState = ["", "", "", "", "", "", "", "", ""];
-function incrementCurrentPlayer(currentPlayer) {
-    currentPlayer++;
-    return currentPlayer;
-}
-function decideTurn(currentPlayer, playerMarker) {
-    if (currentPlayer % 2 == 0) {
-        playerMarker = 'o'
-    } else {
-        playerMarker = 'x'
+
+function decideTurn(currentPlayer) {
+    if (currentPlayer === 'o') {
+        return 'x';
+    }   else  {
+        return  'o';
     }
```

Comment:
Can lose this else as well since you're returning from the if


```js
function decideTurn(currentPlayer) {
    if (currentPlayer === 'o') {
        return 'x';
    } 
    return  'o';
}
```

Reply:
End result is the same but we've avoided using an else needlessly - helps keep things simple.